index.js

```diff
@@ -32,33 +23,29 @@ client.on("message", (message) => {
     message.reply(`Your access code is ${message.guild.id}`);
   } else if (message.content === `${prefix}reset`) {
     FirebaseHelpers.resetDB();
-  } else if (
-    message.content.substr(0, prefix.length + 8) === `${prefix}entries `
-  ) {
+  } else if (message.content.startsWith(`${prefix}entries`)) {
     const entriesFunction = async () => {
       let response = await FirebaseHelpers.numEntries(
         message.guild.name,
-        message.content.substr(prefix.length + 8)
+        message.content.substr(prefix.length + 7)
       );
-      message.reply(response);
+      message.reply(JSON.stringify(response));
     };
     entriesFunction();
   } else if (!message.author.bot) {
```

Comment:
Not a fan of how this is chained on to the if / else chain since it serves a different purpose. The previous two conditions are handlers for the actual message event while this is the sanity check which you put at the end as a failsafe.

It would be more idiomatic to put this check at the start as a single conditional that simply `return` to exit the function if it fails before moving on to the message handler.

And then from there, you could consider doing enums and a switch / case for the bot but then the issue is that you only have two choices and this isn't typescript so you don't have native enums